cpp/src/io/csv/csv_writer.cu

```diff
@@ -234,13 +235,16 @@ gdf_error write_csv(csv_write_arg* args)
     //buffer[memsize] = 0; // just so we can printf if needed
     // now write buffer to file
     // first write the header
-    for( unsigned int idx=0; idx < count; ++idx )
+    if(include_header)
     {
-        gdf_column* col = columns[idx];
-        const char* delim = ((idx+1)<count ? delimiter : terminator);
-        if( col->col_name )
-            filecsv << "\"" << col->col_name << "\"";
-        filecsv << delim;
+        for( unsigned int idx=0; idx < count; ++idx )
+        {
+            gdf_column* col = columns[idx];
+            const char* delim = ((idx+1)<count ? delimiter : terminator);
+            if( col->col_name )
+                filecsv << "\"" << col->col_name << "\"";
```

Comment:
@davidwendt Pandas does not write `"` marks around the column names. Can this be removed or is there a case where we would need this?

Reply:
The quotes are in case the column names have delimiters in them (e.g commas).
I did not want to check through all the column names characters to see if quotes were necessary.

Reply:
Sometimes spaces are used for delimiters and I can see spaces appearing in the column header names too.

Reply:
Thanks! The only potential issue I see is that for column names which are numeric like `1` `2` etc. Would get interpreted as string column names `'1'` in cudf. 

Maybe @kkraus14 can give his opinion on the behavior we would want from cudf.

Reply:
In my opinion we can keep the `"` logic around the headers for now. The edge case of reading numeric column names as string column names seems less extreme than incorrectly interpreting wrong column names due to delimiter clashes. 

In the future if we need to change the behavior it could be something along the lines of checking `if delim in col_name` add `"` around the name else don't